README.md

```diff
@@ -10,8 +10,8 @@
 ![Screenshot](./documentation/screenshot.png)
 
 ## Technologies
-* ASP.NET Core 6
-* RavenDB 5.3
+* ASP.NET Core 8
+* RavenDB 6.0.1
```

Comment:
It's not Angular 14 now?

Reply:
yes - we will update that in separate PR